docs/README.md

```diff
@@ -109,21 +109,15 @@ $ bazel fetch @npm//...
 
 ### Using pnpm workspaces
 
-Here's an example `pnpm-workspace.yaml` file which will automatically discover all packages in the repository, based on the existence of a `package.json` file.
+Here's an example `pnpm-workspace.yaml` file which will find all projects listed under `deeply_nested_project`, `users`, and `experimental`, based on the existence of a `package.json` file.
+
 Make sure to place this file at the root of the repository.
 
 ```yaml
 packages:
-  # Include all directories in the workspace
-  - '*'
-  # Include all subdirectories at any depth
-  - '**/*'
-  # Exclude node_modules folders anywhere in the tree
-  - '!**/node_modules/**'
-  # Exclude node_modules folder in root
-  - '!node_modules'
-  # Exclude anything inside bazel dirs
-  - '!bazel-*/**'
+  - 'deeply_nested_project/**'
```

Comment:
WDYT of `apps/**`, `packages/**` (doublestars), `tools/*` (single star) and `.`?

Reply:
I personally don't prefer to organize top level projects by language or platform, but I know that's more familiar to the community, which will help with adoption. The one caveat is that I don't believe the `.` is necessary.